### PR TITLE
fix(DDM): fix ddm account

### PR DESCRIPTION
**Pull request by houpeng80**
**What this PR does / why we need it**:
  fix ddm account
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
  fix ddm account
```

## PR Checklist

* [ ] Tests added/passed.

```
make testacc TEST=./huaweicloud/services/acceptance/ddm/ TESTARGS='-run TestAccDdmAccount_basic'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ddm/ -v -run TestAccDdmAccount_basic -timeout 360m -parallel 4
=== RUN   TestAccDdmAccount_basic
=== PAUSE TestAccDdmAccount_basic
=== CONT  TestAccDdmAccount_basic
--- PASS: TestAccDdmAccount_basic (872.97s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ddm       873.015s
```

* [ ] Documentation updated.
* [ ] Schema updated.
* [x] CheckDeleted.

  - **a. During query operation (Read Context)**
    aa. Resource not found
   删除了
    ab. Related resources (parent resources) not found
   实例删除了
  - **b. During delete/disassociate/unbind operation (Delete Context)**
    ba. Resource not found
   删除后再删除
    bb. Related resources (parent resources) not found
   实例删了再删除

### CODE REVIEW

File: huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go [hunk 1]
```diff
@@ -95,57 +83,172 @@ func TestAccDdmAccount_basic(t *testing.T) {
...
+
+ availability_zones = [
+   data.huaweicloud_availability_zones.test.names[0]
+ ]
+}
+
+resource "huaweicloud_rds_instance" "test" {
+  name               = "%[2]s"
+  flavor             = "rds.mysql.n1.large.4"
+  security_group_id  = huaweicloud_networking_secgroup.test.id
```
Comment (saf3dfsa) [inline huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go]:
```suggestion
  security_group_id = huaweicloud_networking_secgroup.test.id
```

File: huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go [hunk 2]
```diff
@@ -95,57 +83,172 @@ func TestAccDdmAccount_basic(t *testing.T) {
...
+  subnet_id          = huaweicloud_vpc_subnet.test.id
+  vpc_id             = huaweicloud_vpc.test.id
+
+ availability_zone = [
```
Comment (saf3dfsa) [inline huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go]:
```suggestion
  availability_zone = [
```

File: huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go (hunk 1 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go]: done

File: huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go (hunk 2 above)
Comment (houpeng80) [reply huaweicloud/services/acceptance/ddm/resource_huaweicloud_ddm_account_test.go]: done

**Pull request merged**